shared/templates/rsyslog_logfiles_attributes_modify/bash.template

```diff
@@ -0,0 +1,113 @@
+# platform = Red Hat Virtualization 4,multi_platform_fedora,multi_platform_ol,multi_platform_rhel,multi_platform_sle
+
```

Comment:
nitpick: could you remove this extra empty line, please?

Reply:
:bow: should be handled in [4a89c9d](https://github.com/ComplianceAsCode/content/pull/9789/commits/4a89c9d12328210cc4bfd3ff9f3e9a0977da79a3)